### DISCUSSION TITLE
Agregar ipynb de VS Code a Github

### DISCUSSION THREAD

**Discussion by IdeIsaac** [LaunchX-InnovaccionVirtual/CursoIntroPython#156] [Ideas]
Hola, viajeros. Les quiero compartir como cargué mis archivos ipynb (creo que puede ser cualquier extensión tambien) directo en mi repositorio de Github desde VS Code.

1.- Abrir VS Code

2.- Dirija el cursor a las extensiones y de click, después busque e instale Github y Github Repositories.

3.- Dirja el cursor en el apartado de cuentas y de click e inicie sesión con su cuenta de Github

4.- Le abrirá su navegador, y da click en el botón de continuar. Posiblemente se cargue una segunda página preguntando si quiere que se ejecute una acción,  acceda a ella.
![20220211_174243.jpg](https://user-images.githubusercontent.com/96219383/153687710-8a977113-35f1-47ef-9eaa-f81efbd3c2dc.jpg)

5.- Después de haber iniciado sesión le vuelve a mandar al VS Code con un recuadro, al cual de click en abrir y quedaran vinculado su VS Code.
![20220211_174550.jpg](https://user-images.githubusercontent.com/96219383/153687693-aa152c97-bc5c-4431-a92a-0dc73133d35c.jpg)

6.- Ahora, dirija el cursos al apartado de administrar (engrane en el lado inferior izquierdo) de click, y diríjase paleta de comando.
![20220211_175042.jpg](https://user-images.githubusercontent.com/96219383/153687657-51bc394f-0e1c-4800-afb5-34bce49b5f77.jpg)

7.- observe que en la parte superior del centro, se abre un recuadro y dentro de el escriba Open Remote Repository
![20220211_175132.jpg](https://user-images.githubusercontent.com/96219383/153687629-9abc0d7a-7ed1-46d6-b82a-75b2f3224245.jpg)

8.- Después de escribir lo anterior de click en la opción que aparezca abajo
![20220211_175416.jpg](https://user-images.githubusercontent.com/96219383/153687614-cd9c2eae-1d5a-4ba5-9cde-e5e3a6eb4064.jpg)

9.- Se desplegarán opciones, elija Open Repository from Github
![20220211_175617.jpg](https://user-images.githubusercontent.com/96219383/153687586-b7165f0d-d2d6-42c8-9276-7139b8799c5c.jpg)

10.- Aparecerán los repositorios creados en su cuenta de Github, seleccione en la que desea trabajar y subir los archivos
![20220211_175628.jpg](https://user-images.githubusercontent.com/96219383/153687574-97103e86-0fea-43c9-9844-94a836c61a7c.jpg)

11.- En seguida deberá aparecer VS Code con el explorador abierto y dentro de el los archivos que ha creado (o vacía si no ninguno)
![20220211_175945.jpg](https://user-images.githubusercontent.com/96219383/153687522-36c8df1f-c49d-4fec-aeb6-7384f02cb27a.jpg)

12.- De ahí hasta antes de subir sus archivos utilice VS Code de manera normal.

13.- Para subir sus archivos a Github, diríjase al Git (control de código fuente) y de click
![20220211_180209.jpg](https://user-images.githubusercontent.com/96219383/153687503-7da21511-d2dc-4932-b134-308ed799ca14.jpg)

**14.- Cualquier modificación que se haga desde la carpeta del repositorio aparecerá ahí ** 
![20220211_180404.jpg](https://user-images.githubusercontent.com/96219383/153687469-5b6f3482-d9df-40b7-b107-fe9546d7afa7.jpg)

_15.- Cuando halla terminado su trabajo, en la parte de arriba de donde aparecen los archivos modificados dentro de Git, hay un recuadro en el cual yo, en caso personal escribo un breve comentario de que fue lo que hice, y para subirlos, en la parte de arriba de ese recuadro hay una palomita, el cual al momento de hacer click, confirma y sube las modificaciones directamente a su repositorio de Github _

![20220211_181053.jpg](https://user-images.githubusercontent.com/96219383/153687344-ac77bad1-883f-4787-b8e4-9f9c625d6769.jpg)

Y si te serví de algo, se me ocurre siguiéndome en Github y dando estrellas a mi repo de las katas. Haré lo mismo contigo

**Comment by DanielOe21** [comment]
Excelente aporte!! También como alternativa pueden descargar la Desktop App de Github, ahi mismo pueden clonar los repos a local y trabajar en VS code , después de que se vayan guardando los cambios automáticamente aparecen en la app de escritorio de GitHub y se pueden hacer los commits ! Quizá pronto haga algún tutorial sobre eso igual que el tuyo. Gracias!!

**Reply by IdeIsaac** [reply]
Nos mandas el link de tu tutorial por fa

**Reply by DanielOe21** [reply]
Mañana me encargo de hacerlo!

**Comment by omarfrias-maker** [comment]
Muchas gracias por compartir, me integré esta semana y será de gran ayuda. Gracias.

**Reply by IdeIsaac** [reply]
Excelente y por nada. Si les puedo ayudar ahí me avisan
